packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics-batcher.ts

```diff
@@ -469,4 +468,4 @@ describe('plugin-metrics', () => {
       });
     });
   });
-});
+});
```

Comment:
nitpick: i think we need a new line here

Reply:
Done